libraries/ruby_install.rb

```diff
@@ -96,11 +98,42 @@ def installed?
       ::File.directory?(new_resource.prefix)
     end
 
+    def install_bundler
+      execute = Resource::Execute.new("install bundler", run_context)
+      execute.command("#{new_resource.prefix}/bin/gem install bundler")
+      execute.environment(new_resource.environment)
+      execute.run_action(:run)
+    end
+
     def install_dependencies
       recipe_eval do
         run_context.include_recipe 'build-essential::default'
       end
 
+      # TODO extract to a _common recipe for the common deps per language install
+      if debian?
+        install_package('libxml2-dev')
+        install_package('libxslt-dev')
+        install_package('zlib1g-dev')
+        install_package('ncurses-dev')
+        install_package('libssl-dev')
+      elsif freebsd?
+        install_package('textproc/libxml2')
+        install_package('textproc/libxslt')
+        install_package('devel/ncurses')
+        install_package('libssl-dev')
+      elsif mac_os_x?
+        install_package('libxml2')
+        install_package('libxslt')
+        install_package('libssl-dev')
+      elsif rhel?
+        install_package('libxml2-devel')
+        install_package('libxslt-devel')
+        install_package('ncurses-devel')
+        install_package('zlib-devel')
+        install_package('libssl-devel')
+      end
```

Comment:
Yeah, that is in my TODO at the start of the block. Probably a good call. I was gonna wait until we had a common set and abstract them for all language install recipes.